mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -146,15 +180,39 @@ public RunInfo createRun(CreateRun request) {
    *                     Example : "params.model = 'LogisticRegression' and metrics.acc != 0.9"
    * @param runViewType ViewType for expected runs. One of (ACTIVE_ONLY, DELETED_ONLY, ALL)
```

Comment:
Same here, what does it mean to default to anything?

Reply:
Fair point. If it shouldn't be null, then it should always be defined. Defaulting doesn't make sense.